### PR TITLE
feat(ui5-checkbox): add displayOnly mode

### PR DESCRIPTION
**Pull request by dobrinyonkov**
Closes: #7911

### CODE REVIEW

File: packages/main/src/CheckBox.ts [hunk 1]
```diff
@@ -405,13 +418,27 @@ class CheckBox extends UI5Element implements IFormElement {
...
 	get isCompletelyChecked() {
 		return this.checked && !this.indeterminate;
 	}
 
+	get isDisplayOnly() {
+		return this.displayOnly && !this.disabled;
+	}
+
+	get displayOnlyIcon() {
+		if (this.isCompletelyChecked) {
```
Comment (ilhan007) [inline packages/main/src/CheckBox.ts:433]: I think we only the import "@ui5/webcomponents-icons/dist/accept.js"; icon.
Do we need more icons imports - complete, tri-state, border

Comment (dobrinyonkov) [reply packages/main/src/CheckBox.ts:436]: Yes, I forgot to add them. Thanks!

File: packages/main/test/pages/CheckBox.html [hunk 2]
```diff
@@ -40,6 +40,7 @@
 	<ui5-title>Change Event Test</ui5-title>
 	<ui5-checkbox id="cb1" text="Long long long text"></ui5-checkbox>
 	<ui5-checkbox id="cb2" disabled></ui5-checkbox>
+	<ui5-checkbox id="displayOnlyCb" display-only></ui5-checkbox>
 	<ui5-input id="field"></ui5-input>
```
Comment (plamenivanov91) [inline packages/main/test/pages/CheckBox.html:44]: can we add all of its variants for easier theme checking like this

	<ui5-checkbox display-only checked></ui5-checkbox>
	<ui5-checkbox display-only indeterminate checked></ui5-checkbox>

**Review by plamenivanov91 — CHANGES_REQUESTED** [review]
The newly added display only type doesn't appear good on darker themes (dark and hcb).

It should be brighter.

File: packages/main/test/pages/CheckBox.html (hunk 2 above)
Comment (dobrinyonkov) [reply packages/main/test/pages/CheckBox.html:47]: That makes sense, Thanks!

**Comment by dobrinyonkov** [comment]
> The newly added display only type doesn't appear good on darker themes (dark and hcb).
> 
> It should be brighter.

Fixed, thanks for catching that.

**Review by plamenivanov91 — APPROVED** [review]

**Pull request merged**